README.md

```diff
@@ -213,6 +213,27 @@ in place of a file being edited.)
 /usr/include/dmd/phobos/std/conv.d  f   9494
 ```
 
+## Find usage of symbol at cursor
+```dcd-client --localUsage -c 123```
+
+The "--localUsage" or "-u" flags cause the client to instruct the server
```

Comment:
Indeed so i'll go for "use" but i have to keep the old switch for compatibility with my IDE (let's say for 2 updates).